Don't care whatever libdir they think we should be using, we know best!

--- configure.in
+++ configure.in
@@ -2741,6 +2741,7 @@
         wx_cv_std_libpath="lib";
         ;;
 esac
+wx_cv_std_libpath="@GENTOO_PORTAGE_LIBDIR@"
 
 AC_MSG_RESULT($wx_cv_std_libpath)
 
